db/migrate/20160816144624_create_damages.rb

```diff
@@ -0,0 +1,14 @@
+class CreateDamages < ActiveRecord::Migration
+  def change
+    create_table :damages do |t|
+      t.string :uuid
+      t.string :damage_issued_reservation_uuid
```

Comment:
These uuids do exist in the app. When the damage create, the uuid from the rental table of the app will pass to the api for "damage_issued_reservation_uuid". For the "damage_fixed_reservation_uuid", it would come from the "Reservation(we are going to change the name to Repair or something)" table's uuid. Would that be make sense?